src/site/xdoc/filters/suppresswithnearbycommentfilter.xml

```diff
@@ -245,7 +253,8 @@ public class Example5 {
         <div class="wrapper"><pre class="prettyprint"><code class="language-java">
 public class Example6 {
   // @cs.suppress [ConstantName|NoWhitespaceAfter] A comment here
-  public static final int [] array = {};
+  public static final int [] array = {}; // filtered violation
+  // filtered violation above
```

Comment:
this is right way to address two violations on same line, please add messages or explain more clearly what stoppping you to make it.